src/Masterserver/Config.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
+------------------------------------------------------------------------------
+-- |
+-- Copyright 2016-2016 the openage authors. See copying.md for legal info.
+-- Module: Masterserver.Config
+--
+-- This Module defines the masterservers config parser.
+
+------------------------------------------------------------------------------
+module Masterserver.Config
+       ( getPostgresConf
+       , getPort
+       , getVersion
+       ) where
+
+import Data.Aeson.TH
+import Data.Yaml
+import Database.Persist.Postgresql
+
+-- | Config datatype used for parsing.
+data Config = Config
+  { acceptedVersion :: ![Int] -- ^ Client version accepted by server
+  , port :: !Int              -- ^ Port to run server on
+  , database :: !Value        -- ^ Database connection info
+  } deriving Show
+
+-- | Derives aeson toJSON and fromJSON instances for Config.
+$(deriveJSON defaultOptions ''Config)
+
+-- | Get PostgresConf from yaml file.
+getPostgresConf :: IO PostgresConf
+getPostgresConf = do
+  Just yaml <- decodeFile "etc/openage/masterserver.yaml"
```

Comment:
Can we add a command line option that specifies the config location? Which has a default value of `/etc/openage/masterserver.yaml` so the systemd service file works without setting the location?